src/js/view/controls/controls.js

```diff
@@ -238,6 +238,8 @@ export default class Controls {
                 // Let event bubble upwards
                 return true;
             }
+            const menuVisible = this.settingsMenu.visible;
```

Comment:
All the `if` statements are using `!menuVisible`. Might as well flip this to `menuHidden` or something of the sort.